tardis/io/atomic.py

```diff
@@ -38,270 +37,106 @@ def tests_data_path(fname):
-
-    Returns
-    -------
-
-    data : `~astropy.table.Table`
-        table with fields wavelength, atomic_number, ion_number, f_ul, f_lu, level_id_lower, level_id_upper.
-    """
-
-    data_table = read_hdf5_data(fname, 'lines_data')
-    #data_table.columns['ionization_energy'].convert_units_to('erg')
-
-    return data_table
-
-
-def read_zeta_data(fname):
-    """
-    This function reads the recombination coefficient data from the HDF5 file
-
-
-    :return:
-    """
-
-    if fname is None:
-        raise ValueError('fname can not be "None" when trying to use NebularAtom')
-
-    if not os.path.exists(fname):
-        raise IOError('HDF5 File doesn\'t exist')
-
-    h5_file = h5py.File(fname, 'r')
-
-    if 'zeta_data' not in h5_file.keys():
-        raise ValueError('zeta_data not available in this HDF5-data file. It can not be used with NebularAtomData')
-
-    zeta_data = h5_file['zeta_data']
-    t_rads = zeta_data.attrs['t_rad']
-    return pd.DataFrame(zeta_data[:,2:], index=pd.MultiIndex.from_arrays(zeta_data[:,:2].transpose().astype(int)),
-                 columns=t_rads)
-
-
-def read_collision_data(fname):
-    if fname is None:
-        raise ValueError('fname can not be "None" when trying to use NebularAtom')
-
-    if not os.path.exists(fname):
-        raise IOError('HDF5 File doesn\'t exist')
-
-    h5_file = h5py.File(fname, 'r')
-
-    if 'collision_data' not in h5_file.keys():
-        raise ValueError('collision_data not available in this HDF5-data file. It can not be used with NLTE')
-
-    collision_data = np.array(h5_file['collision_data'])
-    collision_temperatures = h5_file['collision_data'].attrs['temperatures']
-
-    return collision_data, collision_temperatures
-
-
-def read_ion_cx_data(fname):
-    try:
-        h5_file = h5py.File(fname, 'r')
-        ion_cx_th_data = h5_file['ionization_cx_threshold']
-        ion_cx_sp_data = h5_file['ionization_cx_support']
-        return ion_cx_th_data, ion_cx_sp_data
-    except IOError, err:
-        print(err.errno)
-        print(err)
-        logger.critical('Cannot import. Error opening the file to read ionization_cx')
-
-
-def read_macro_atom_data(fname):
-    if fname is None:
-        raise ValueError('fname can not be "None" when trying to use NebularAtom')
-
-    if not os.path.exists(fname):
-        raise IOError('HDF5 File doesn\'t exist')
-
-    h5_file = h5py.File(fname, 'r')
-
-    if 'macro_atom_data' not in h5_file.keys():
-        raise ValueError('Macro Atom Data (macro_atom_data) is not in this HDF5-data file. '
-                         'It is needed for complex line interaction')
-    macro_atom_data = h5_file['macro_atom_data']
-
-    macro_atom_counts = h5_file['macro_atom_references']
-
-    return macro_atom_data, macro_atom_counts
-
-
 class AtomData(object):
     """
     Class for storing atomic data
 
-    AtomData
-    ---------
-
     Parameters
     ----------
```

Comment:
@wkerzendorf @yeganer  I used the legacy names for the Dataframes but I suggest changing `atom_data` -> `atom_masses`, `ionization_data` -> `ionization_energies` . Not sure about other dataframes.